packages/core/src/agent/AgentConfig.ts

```diff
@@ -191,8 +191,16 @@ export class AgentConfig {
   public toJSON() {
     return {
       ...this.initConfig,
-      logger: this.logger !== undefined,
-      agentDependencies: this.agentDependencies != undefined,
+      walletConfig: {
+        ...this.walletConfig,
+        key: this.walletConfig?.key ? '[omitted...]' : undefined,
+        storage: {
+          ...this.walletConfig?.storage,
+          credentials: this.walletConfig?.storage?.credentials ? '[omitted...]' : undefined,
```

Comment:
This is great and long awaited addition 👍 

Just a naming idea... wouldn't it be better to use rather `[hidden...]` or, quite common use of `*****` (I saw it in Github Action I think)

Reply:
👍🏾